src/App.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './App.css';
+
+function App() {
+    return <div className="App">а</div>;
```

Comment:
It will be better if you remove the 'a' letter in the div element, if you don't necessary it, of course.